### PR TITLE
Testing environment setup

### PR DESCRIPTION
**Pull request by 08martinm**
Installed mocha and chai. 
Created test file. 
Set up mocha to run in Docker by updating Dockerfile.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "A reputation-based, transaction-oriented platform",
   "main": "index.js",
   "scripts": {
-    "start": "webpack --progress --colors --watch & nodemon server/server.js",
+    "start": "webpack --progress --colors --watch & nodemon server/index.js",
     "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (cassiebaer) [inline package.json:8]: We could change this to `mocha test` so we can easily run `dc run web npm test` to run all the tests

**Comment by cassiebaer** [comment]
Also, where is Travis??

**Comment by cassiebaer** [comment]
:ship:

**Comment by donovanCarreno** [comment]
:shipit:

**Pull request merged**